stdlib/ballerina-http/src/main/ballerina/ballerina/net/http/natives.bal

```diff
@@ -69,12 +90,6 @@ public native function <InRequest req> setEntity (mime:Entity entity);
 @Return {value:"The map of query params"}
 public native function <InRequest req> getQueryParams () (map);
 
-@Description {value:"Retrieves the named property from the request"}
-@Param {value:"req: The inbound request message"}
-@Param {value:"propertyName: The name of the property"}
-@Return {value:"The property value"}
-public native function <InRequest req> getProperty (string propertyName) (string);
```

Comment:
Are we only removing inRequest "getProperty"? What about OutRequest and Response structs?